docs/core/versions/selection.md

```diff
@@ -36,6 +36,10 @@ On rare occasions, you may need to use an earlier version of the SDK. You specif
 
 *global.json* can be placed anywhere in the file hierarchy. The CLI searches upward from the project directory for the first *global.json* it finds. You control which projects a given *global.json* applies to by its place in the file system. The .NET CLI searches for a *global.json* file iteratively navigating the path upward from the current working directory. The first *global.json* file found specifies the version used. If that version is installed, that version is used. If the SDK specified in the *global.json* is not found, the .NET CLI rolls forward to the latest SDK installed. Roll-forward is the same as the default behavior, when no *global.json* file is found.
 
+> [!TIP]
+> We recommend that you add a *global.json* file to the root of your drive set to the latest **supported** SDK version.
+> That way, you ensure that all your projects will use the latest **supported** SDK version by default and not the latest **installed** version, which might be a preview version. Whenever a new supported version is released, then you can just edit that file to update the SDK version.
```

Comment:
@mairaw two more observations:
- "...to the latest **supported** SDK version." I think it should be noted that the specified version must be installed. In particular, the last sentence — "Whenever a new supported version is released, then you can just edit that file to update the SDK version." — might be understood as if one doesn't need to **install** the released SDK version, but only update the *global.json* file and that mere edit would "update the SDK version". What about:
  > Whenever you install a newer supported SDK, edit that file to specify a new SDK version.

- And this is nit-picking squared: "all your projects will use the latest **supported** SDK version by default and not the latest **installed** version" — it doesn't compute if the latest supported SDK version **is** the latest installed version. I guess, the main motivation for the tip is to combat default using of preview versions, so can the sentence be shortened?
  > That way, you ensure that all your projects will use the latest **supported** SDK version by default and not a preview version.

  Again, to my understanding, it's crucial that that latest supported SDK version is *installed*. Otherwise, CLI fallbacks to the latest installed which might be a preview?

Reply:
+1 to Livar's comment, we should not be recommending this approach to customers in docs. I suspect there was a misunderstanding on Scott's part if he suggested this.

Reply:
Sounds good @Andrew-MSFT and @livarcocc. I'm closing this.